File: evals/fixtures/ai-tell-cleanup/05-verbose-docstring/input.java
package com.example;

public interface UserRepository {

    /**
     * ID로 사용자를 조회합니다.
     *
     * @param id 조회할 사용자의 ID입니다. null이 아니어야 합니다.
     * @return 조회된 User 객체를 반환합니다. 존재하지 않는 경우 null을 반환합니다.
     */
    User findById(Long id);

    /**
     * 사용자를 저장합니다.
     *
     * @param user 저장할 User 객체입니다. null이 아니어야 합니다.
     * @return 저장된 User 객체를 반환합니다.
     * @throws IllegalArgumentException user가 null인 경우 발생합니다.
     */
    User save(User user);

    /**
     * 모든 사용자 목록을 반환합니다.
     *
     * @return 모든 User 객체의 리스트를 반환합니다.
     */
    List<User> findAll();
}

File: evals/fixtures/ai-tell-cleanup/02-code-restating-comment/input.kt
package com.example

import org.springframework.stereotype.Service

@Service
class UserService(private val repo: UserRepository) {

    fun findById(id: Long): User? {
        // ID로 사용자를 조회한다
        val user = repo.findById(id).orElse(null)
        // 결과를 반환한다
        return user
    }

    fun save(user: User): User {
        // 유저를 저장한다
        return repo.save(user)
    }

    fun deleteById(id: Long) {
        // 해당 ID의 유저를 삭제한다
        repo.deleteById(id)
    }
}
